docs/framework/whats-new/index.md

```diff
@@ -1711,7 +1711,7 @@ With Ngen PDBs, NGen can create a PDB that contains the IL-to-native mapping wit
 
     Windows 10 includes a new high-scalability networking algorithm that makes better use of machine resources by reusing local ports for outbound TCP connections. .NET Framework 4.6 supports the new algorithm, enabling .NET apps to take advantage of the new behavior. In previous versions of Windows, there was an artificial concurrent connection limit (typically 16,384, the default size of the dynamic port range), which could limit the scalability of a service by causing port exhaustion when under load.
 
-    In the .NET Framework 4.6, two new APIs have been added to enable port reuse, which effectively removes the 64K limit on concurrent connections:
+    In .NET Framework 4.6, two new APIs have been added to enable port reuse, which effectively removes the 64-K limit on concurrent connections:
```

Comment:
should this be KB instead? https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=26920

```suggestion
    In .NET Framework 4.6, two APIs have been added to enable port reuse, which effectively removes the 64 KB limit on concurrent connections:
```